upload-chunked-file.js

```diff
@@ -0,0 +1,43 @@
+function chunkFileUpload(file, serverUrl, chunkSize) {
```

Comment:
Consider adding input validation to check that `file`, `serverUrl`, and `chunkSize` have valid and expected values to prevent runtime errors.